networkx/drawing/nx_pylab.py

```diff
@@ -639,8 +639,7 @@ def draw_networkx_edges(
 
     if edgelist is None:
         edgelist = list(G.edges())
-
-    if not edgelist or len(edgelist) == 0:  # no edges!
+    elif len(edgelist) == 0:  # no edges!
```

Comment:
Ack -- sorry I didn't see this before.  

I think this ```elif``` should be ```if``` because even after the previous if clause, edgelist could be an empty list (if the graph has no edges).

Reply:
Ah, you're right, fixed